docs/docs/configuration/monitoring.md

```diff
@@ -0,0 +1,15 @@
+# Monitoring
+## Ensure all topic partitions are assigned to consumer group in subscription
```

Comment:
Missing articles: `Ensure all topic partitions are assigned to a consumer group in a subscription`